<!DOCTYPE html>
<html lang="zh-CN">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<meta name="theme-color" content="#222">
<meta name="generator" content="Hexo 5.4.0">
  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon-next.png">
  <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32-next.png">
  <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16-next.png">
  <link rel="mask-icon" href="/images/logo.svg" color="#222">

<link rel="stylesheet" href="/css/main.css">

<link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:300,300italic,400,400italic,700,700italic|Roboto Slab:300,300italic,400,400italic,700,700italic&display=swap&subset=latin,latin-ext">
<link rel="stylesheet" href="/lib/font-awesome/css/all.min.css">

<script id="hexo-configurations">
    var NexT = window.NexT || {};
    var CONFIG = {"hostname":"example.com","root":"/","scheme":"Pisces","version":"7.8.0","exturl":false,"sidebar":{"position":"left","display":"post","padding":18,"offset":12,"onmobile":false},"copycode":{"enable":false,"show_result":false,"style":null},"back2top":{"enable":true,"sidebar":false,"scrollpercent":false},"bookmark":{"enable":false,"color":"#222","save":"auto"},"fancybox":false,"mediumzoom":false,"lazyload":false,"pangu":false,"comments":{"style":"tabs","active":null,"storage":true,"lazyload":false,"nav":null},"algolia":{"hits":{"per_page":10},"labels":{"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}},"localsearch":{"enable":false,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false},"motion":{"enable":true,"async":false,"transition":{"post_block":"fadeIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}}};
  </script>

  <meta property="og:type" content="article">
<meta property="og:title" content="单细胞转录组文献分析方法[三]: Single-cell transcriptomic analysis of Alzheimer&#39;s disease">
<meta property="og:url" content="http://example.com/2021/12/04/%E5%8D%95%E7%BB%86%E8%83%9E%E8%BD%AC%E5%BD%95%E7%BB%84%E6%96%87%E7%8C%AE%E5%88%86%E6%9E%90%E6%96%B9%E6%B3%95-%E4%B8%89-Single-cell-transcriptomic-analysis-of-Alzheimer-disease/index.html">
<meta property="og:site_name" content="TypeError">
<meta property="og:locale" content="zh_CN">
<meta property="og:image" content="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/image-20211204160946331.png">
<meta property="og:image" content="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig1_HTML.png">
<meta property="og:image" content="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig1_HTML-20211204163924346.png">
<meta property="og:image" content="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig2_HTML.png">
<meta property="og:image" content="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig3_HTML.png">
<meta property="og:image" content="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig4_HTML.png">
<meta property="og:image" content="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig6_ESM.jpg">
<meta property="og:image" content="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig7_ESM.jpg">
<meta property="og:image" content="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig8_ESM.jpg">
<meta property="og:image" content="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig9_ESM.jpg">
<meta property="og:image" content="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig11_ESM.jpg">
<meta property="og:image" content="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig12_ESM.jpg">
<meta property="article:published_time" content="2021-12-04T08:03:33.000Z">
<meta property="article:modified_time" content="2021-12-04T08:56:31.544Z">
<meta property="article:author" content="Max">
<meta property="article:tag" content="single-cell RNA-seq">
<meta property="article:tag" content="brain">
<meta property="article:tag" content="prefrontal cortex">
<meta property="article:tag" content="disease">
<meta property="article:tag" content="AD">
<meta name="twitter:card" content="summary">
<meta name="twitter:image" content="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/image-20211204160946331.png">

<link rel="canonical" href="http://example.com/2021/12/04/%E5%8D%95%E7%BB%86%E8%83%9E%E8%BD%AC%E5%BD%95%E7%BB%84%E6%96%87%E7%8C%AE%E5%88%86%E6%9E%90%E6%96%B9%E6%B3%95-%E4%B8%89-Single-cell-transcriptomic-analysis-of-Alzheimer-disease/">


<script id="page-configurations">
  // https://hexo.io/docs/variables.html
  CONFIG.page = {
    sidebar: "",
    isHome : false,
    isPost : true,
    lang   : 'zh-CN'
  };
</script>

  <title>单细胞转录组文献分析方法[三]: Single-cell transcriptomic analysis of Alzheimer's disease | TypeError</title>
  






  <noscript>
  <style>
  .use-motion .brand,
  .use-motion .menu-item,
  .sidebar-inner,
  .use-motion .post-block,
  .use-motion .pagination,
  .use-motion .comments,
  .use-motion .post-header,
  .use-motion .post-body,
  .use-motion .collection-header { opacity: initial; }

  .use-motion .site-title,
  .use-motion .site-subtitle {
    opacity: initial;
    top: initial;
  }

  .use-motion .logo-line-before i { left: initial; }
  .use-motion .logo-line-after i { right: initial; }
  </style>
</noscript>

</head>

<body itemscope itemtype="http://schema.org/WebPage">
  <div class="container use-motion">
    <div class="headband"></div>

    <header class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-container">
  <div class="site-nav-toggle">
    <div class="toggle" aria-label="切换导航栏">
      <span class="toggle-line toggle-line-first"></span>
      <span class="toggle-line toggle-line-middle"></span>
      <span class="toggle-line toggle-line-last"></span>
    </div>
  </div>

  <div class="site-meta">

    <a href="/" class="brand" rel="start">
      <span class="logo-line-before"><i></i></span>
      <h1 class="site-title">TypeError</h1>
      <span class="logo-line-after"><i></i></span>
    </a>
  </div>

  <div class="site-nav-right">
    <div class="toggle popup-trigger">
    </div>
  </div>
</div>




<nav class="site-nav">
  <ul id="menu" class="main-menu menu">
        <li class="menu-item menu-item-home">

    <a href="/" rel="section"><i class="fa fa-home fa-fw"></i>首页</a>

  </li>
        <li class="menu-item menu-item-tags">

    <a href="/tags/" rel="section"><i class="fa fa-tags fa-fw"></i>标签</a>

  </li>
        <li class="menu-item menu-item-categories">

    <a href="/categories/" rel="section"><i class="fa fa-th fa-fw"></i>分类</a>

  </li>
        <li class="menu-item menu-item-archives">

    <a href="/archives/" rel="section"><i class="fa fa-archive fa-fw"></i>归档</a>

  </li>
        <li class="menu-item menu-item-schedule">

    <a href="/schedule/" rel="section"><i class="fa fa-calendar fa-fw"></i>日程表</a>

  </li>
  </ul>
</nav>




</div>
    </header>

    
  <div class="back-to-top">
    <i class="fa fa-arrow-up"></i>
    <span>0%</span>
  </div>


    <main class="main">
      <div class="main-inner">
        <div class="content-wrap">
          

          <div class="content post posts-expand">
            

    
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="http://example.com/2021/12/04/%E5%8D%95%E7%BB%86%E8%83%9E%E8%BD%AC%E5%BD%95%E7%BB%84%E6%96%87%E7%8C%AE%E5%88%86%E6%9E%90%E6%96%B9%E6%B3%95-%E4%B8%89-Single-cell-transcriptomic-analysis-of-Alzheimer-disease/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="Max">
      <meta itemprop="description" content="在学了在学了">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="TypeError">
    </span>
      <header class="post-header">
        <h1 class="post-title" itemprop="name headline">
          单细胞转录组文献分析方法[三]: Single-cell transcriptomic analysis of Alzheimer's disease
        </h1>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>
              

              <time title="创建时间：2021-12-04 16:03:33 / 修改时间：16:56:31" itemprop="dateCreated datePublished" datetime="2021-12-04T16:03:33+08:00">2021-12-04</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E5%8D%95%E7%BB%86%E8%83%9E%E8%BD%AC%E5%BD%95%E7%BB%84/" itemprop="url" rel="index"><span itemprop="name">单细胞转录组</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E5%8D%95%E7%BB%86%E8%83%9E%E8%BD%AC%E5%BD%95%E7%BB%84/%E6%96%87%E7%8C%AE/" itemprop="url" rel="index"><span itemprop="name">文献</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
        <p><img src="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/image-20211204160946331.png" alt="image-20211204160946331"></p>
<p><img src="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig1_HTML.png" alt="Fig. 1"></p>
<span id="more"></span>

<h1 id="Abstract"><a href="#Abstract" class="headerlink" title="Abstract"></a>Abstract</h1><p>Alzheimer’s disease is a pervasive neurodegenerative disorder, the molecular complexity of which remains poorly understood. Here, we analysed 80,660 single-nucleus transcriptomes from the prefrontal cortex of 48 individuals with varying degrees of Alzheimer’s disease pathology. Across six major brain cell types, we identified transcriptionally distinct subpopulations, including those associated with pathology and characterized by regulators of myelination, inflammation, and neuron survival. The strongest disease-associated changes appeared early in pathological progression and were highly cell-type specific, whereas genes upregulated at late stages were common across cell types and primarily involved in the global stress response. Notably, we found that female cells were overrepresented in disease-associated subpopulations, and that transcriptional responses were substantially different between sexes in several cell types, including oligodendrocytes. Overall, myelination-related processes were recurrently perturbed in multiple cell types, suggesting that myelination has a key role in Alzheimer’s disease pathophysiology. Our single-cell transcriptomic resource provides a blueprint for interrogating the molecular and cellular basis of Alzheimer’s disease.</p>
<h1 id="Methods"><a href="#Methods" class="headerlink" title="Methods"></a>Methods</h1><h2 id="Cell-clustering"><a href="#Cell-clustering" class="headerlink" title="Cell clustering"></a>Cell clustering</h2><p>All <strong>75,060 cells</strong> were combined into a single dataset. <strong>Normalization and clustering were done with the <mark>SCANPY</mark> package.</strong> In brief, counts for all nuclei were scaled by the total library size multiplied by 10,000, and transformed to log space. </p>
<p>A total of 3,188 highly variable genes were identified based on dispersion and mean, the technical influence of the total number of counts was regressed out, and the values were rescaled. These preprocessing steps were performed by sequentially using the functions <code>normalize_per_cell</code>, <code>filter_genes_dispersion</code>, <code>log1p.regress_out</code>, and <code>scale</code> in SCANPY. </p>
<p><strong>Principal component analysis</strong> (PCA) was performed on the variable genes, and <em>t</em>-SNE was run on the top 10 principal components (PCs) using the <strong>Multicore-TSNE</strong> package (<a target="_blank" rel="noopener" href="https://github.com/DmitryUlyanov/Multicore-TSNE">https://github.com/DmitryUlyanov/Multicore-TSNE</a>). The top 50 PCs were used to build a <em>k</em>-nearest-neighbours cell–cell graph with <em>k</em> = 30 neighbours; subsequently spectral decomposition over the graph was performed with 15 components, and the Louvain graph-clustering algorithm was applied to identify cell clusters. These analyses were performed using the functions <code>pca</code>, <code>neighbours</code>, and <code>louvain</code> in SCANPY. </p>
<p>We confirmed that a number of PCs greater than 30 captures 100% of the variance of the data. The initial pre-clustering analysis resulted in <strong>20 pre-clusters with a median number of 2,990 cells, ranging from 413 to 15,900 cells</strong>, after excluding 2 pre-clusters of 360 and 791 cells that <strong>reflected low-quality cells</strong> (<strong>that is, cells that showed mixed cell-type markers; extreme complexity with many more genes expressed than other cells; either too many or too few reads; and, in one case, cells isolated almost exclusively from one individual</strong>). For each pre-cluster, differentially expressed genes were detected using the variance-adjusted <em>t</em>-test as implemented in the function <code>rank_genes_groups</code> in SCANPY. </p>
<p>The top 500 ranking genes were extracted for each cluster, and used to test for overlap with markers as previously reported. The same clustering protocol was used for both pre- and sub-clustering analyses. During sub-clustering, additional potentially spurious clusters representing low-quality or doublet cells were detected on the basis of extreme separation from the rest of the sub-clusters from the same cell type. Of these, those having a distinctly high number of total counts and mixed expression of markers from different cell types were tagged as potential doublets and not considered for downstream analyses, <strong>resulting in a total of 70,634 cells.</strong></p>
<h2 id="Cell-type-annotation-and-sub-clustering"><a href="#Cell-type-annotation-and-sub-clustering" class="headerlink" title="Cell type annotation and sub-clustering"></a>Cell type annotation and sub-clustering</h2><p>For each pre-cluster, we assigned a cell-type label using statistical enrichment for sets of marker genes and manual evaluation of gene expression for small sets of known marker genes. Enrichment was statistically assessed using the hypergeometric distribution (Fisher’s exact test) and FDR correction over all gene sets and pre-clusters. Broad cell-type clusters were defined by grouping together all pre-clusters corresponding to the same cell type. Sub-clustering analysis was performed independently over each broad cell-type cluster.</p>
<h2 id="Marker-identification"><a href="#Marker-identification" class="headerlink" title="Marker identification"></a>Marker identification</h2><p>For sub-clusters, a set of markers (specifically overexpressed) genes was defined by a differential expression analysis of the cells grouped in each sub-cluster against the remaining cells within the corresponding broad cell-type cluster. This analysis was applied to all cell types independently. Significantly overexpressed genes were defined based on the Wilcoxon rank-sum test with a FDR-corrected <em>P</em> value ≤ 0.01 and a log2(mean gene expression across cells in sub-cluster/mean gene expression across cells in other sub-clusters) of 0.5. Only genes detected in at least 25% of the cells within the given sub-cluster were considered.</p>
<p>Gene Ontology enrichment analyses were performed using the R package <mark>gProfileR</mark> (<a target="_blank" rel="noopener" href="https://cran.r-project.org/web/packages/gProfileR/index.html">https://cran.r-project.org/web/packages/gProfileR/index.html</a>) and using the <em>P</em> value-ranked gene lists as input. For all cases we used the set of 17,926 protein-coding genes included in the quality control data as background. </p>
<h2 id="Differential-gene-expression-analysis"><a href="#Differential-gene-expression-analysis" class="headerlink" title="Differential gene-expression analysis"></a>Differential gene-expression analysis</h2><p><mark><strong>snRNAseq-based differential expression analysis</strong></mark> was assessed using two tests. </p>
<ul>
<li>First, a cell-level analysis was performed using the Wilcoxon rank-sum test and FDR multiple-testing correction. </li>
<li>Second, a Poisson mixed model accounting for the individual of origin for nuclei and for unwanted sources of variability was performed using the R packages <strong>lme4</strong> and <strong>RUV-seq</strong>, respectively. </li>
</ul>
<p>The consistency of DEGs detected using the cell-level analysis model with those obtained with the Poisson mixed model was assessed by comparing the directionality and rank of DEGs in the two models. Consistency in directionality for all cell types was measured by counting the fraction of the top 1,000 DEGs (ranked by FDR scores) detected in cell-level analysis that showed consistent direction in the mixed model. High consistency was found, with a median fraction of 0.99. Global consistency between the two models was assessed statistically using a resampling test. We tested whether the differential <em>P</em> value and <em>z</em>-score ranks corresponding to genes detected as upregulated or downregulated in the cell-level analysis were significantly higher or lower than those that were expected by chance when computed using the mixed model. Expected scores were estimated by randomly sampling same-sized gene sets (<em>n</em> = 1,000 replicates). Both significance rank and direction deviated significantly from expectation, with directionality consistent for up- or downregulated genes. Results from the consistency tests and from both the cell-level and mixed-model differential tests are included in Supplementary Table <a target="_blank" rel="noopener" href="https://www.nature.com/articles/s41586-019-1195-2#MOESM4">2</a>. For analyses involving DEG counts, only genes that were significantly supported by both models using the criteria FDR-corrected <em>P</em> &lt; 0.01 in a two sided Wilcoxon-rank sum test, absolute log2(mean gene expression in AD category *x*/mean gene expression in AD category *y*) &gt; 0.25, and FDR-corrected <em>P</em> &lt; 0.05 in a Poisson mixed model were considered. Owing to their low cell counts, pericyte and endothelial cell populations were excluded from differential analyses.</p>
<p><mark><strong>Bulk RNA-seq differential analysis</strong></mark> was performed by fitting a linear model using the R package <strong>limma</strong>, accounting for the covariates age, RNA integrity number, post-mortem interval, and plate batches. The pathological definition of AD groups provided by ROSMAP was used. This classification defines AD or non-AD groups based exclusively on the overall pathological burden, without considering clinical diagnosis. We used the labels AD-pathology and no-pathology for consistency.</p>
<p>Consistency of gene-expression perturbations in the different cell types observed in snRNA-seq data with those detected in tissue-level bulk RNA-seq was assessed using a resampling approach. To test whether the genes identified as DEGs in single-cell data were also detected as high ranking in the differential analysis in bulk data, a <em>z</em>-score statistic was computed to quantify the deviation of the observed differential (<em>P</em> value) rank scores obtained in the bulk analysis for the genes detected as DEGs in single cells, relative to those observed in 1,000 randomly chosen gene sets. This analysis was performed for each cell type independently.</p>
<h1 id="Figure"><a href="#Figure" class="headerlink" title="Figure"></a>Figure</h1><h2 id="Fig-1-Cell-type-specific-gene-expression-changes-in-AD-pathology"><a href="#Fig-1-Cell-type-specific-gene-expression-changes-in-AD-pathology" class="headerlink" title="Fig. 1: Cell-type-specific gene-expression changes in AD pathology."></a>Fig. 1: Cell-type-specific gene-expression changes in AD pathology.</h2><p><img src="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig1_HTML-20211204163924346.png" alt="Fig. 1"></p>
<blockquote>
<p><strong>a</strong>, Genes most upregulated: excitatory (Ex) and inhibitory (In) neurons, astrocytes (Ast), oligodendrocytes (Oli), oligodendrocyte precursor cells (Opc), and microglia (Mic). <strong>b</strong>, DEG counts for each cell type (two-sided Wilcoxon rank-sum test, FDR &lt; 0.01, log2(mean gene expression in AD-pathology/mean gene expression in no-pathology) &gt; 0.25, Poisson mixed-model FDR &lt; 0.05). The intensity of the blue colour and the size of the squares are proportional to entry values. **c**, RT–qPCR validation. snRNA-seq differential scores for excitatory- and inhibitory-neuron DEGs (*z*-score, Poisson mixed model) (left) and qPCR validation (right). NeuN-positive nuclei were isolated by fluorescence-activated cell sorting (*n* = 8 AD-pathology individuals and *n* = 8 no-pathology individuals). Data are mean ± s.e.m.; ****P* &lt; 0.001, ***P* &lt; 0.01, **P* &lt; 0.05; NS, *P* &gt; 0.05 (Student’s two-tailed <em>t</em>-test). <strong>d</strong>, <strong>e</strong>, Differential scores (<em>z</em>-scores, Poisson mixed model) for the top DEGs from snRNA-seq (<strong>d</strong>), and the corresponding values from bulk RNA-seq (<strong>e</strong>) (ROSMAP cohorts; <em>n</em> = 484, <em>P</em> &lt; 0.01). <strong>f</strong>, Global consistency analysis of snRNA-seq versus bulk RNA-seq. Agreement or disagreement was estimated by deviation from random expectation (<em>z</em>-score) of single-cell DEG average rank scores in the ranked list of genes from bulk differential analysis. <strong>g</strong>, Differential expression of <em>APOE</em> in microglia (<em>n</em> = 955 cells, 24 AD-pathology individuals; <em>n</em> = 965 cells, 24 no-pathology individuals) and astrocytes (<em>n</em> = 1,830 cells, 24 AD-pathology individuals; <em>n</em> = 1,562 cells, 24 no-pathology individuals; <em>z</em>-score, Poisson mixed model; **<em><em>P</em> = 0.001, **P</em> = 0.05, corresponding <em>P</em> value calculated from two-sided, standard normal distribution). <strong>h</strong>, Binary plots indicating with bars whether a gene (column) is a DEG in a given cell type (rows) or not (<em>n</em> = 1,031 DEGs). Right, six genes associated with myelination and/or axon regeneration.</p>
</blockquote>
<h2 id="Fig-2-Gene-expression-changes-in-the-progression-of-AD-pathology"><a href="#Fig-2-Gene-expression-changes-in-the-progression-of-AD-pathology" class="headerlink" title="Fig. 2: Gene-expression changes in the progression of AD pathology."></a>Fig. 2: Gene-expression changes in the progression of AD pathology.</h2><p><img src="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig2_HTML.png" alt="Fig. 2"></p>
<blockquote>
<p><strong>a</strong>, Phenotypic clustering of 48 individuals (columns) using clinico-pathological variables (rows) measuring neuronal neurofibrillary tangle density (tangles), neurofibrillary tangle burden (NFT), global AD-pathology burden (gpath), global measure of neocortical pathology (gpath_3neocort), neuritic plaque burden (plaq_n), overall amyloid level (amyloid), and global cognitive function (last valid score) (cogn_global_lv). <strong>b</strong>, Progressive changes in DEG counts for each cell type for early and late AD-pathology individuals (two-sided Wilcoxon rank-sum test, FDR &lt; 0.01, log2(mean gene expression in early-pathology/mean gene expression in no-pathology) &gt; 0.25 or log2(mean gene expression in late-pathology/mean gene expression in early-pathology) and similarly for the opposite direction, Poisson mixed-model FDR &lt; 0.05). The intensity of the blue colour and the size of the squares are proportional to entry values. <strong>c</strong>, Most-significantly altered (based on <em>P</em> value rank) genes (rows) for each cell type (columns) and comparison. Column-scaled <em>z</em>-scores computed with a Poisson mixed model are shown (FDR &lt; 0.01, two-sided Wilcoxon rank-sum test). <strong>d</strong>, Fraction of total upregulated genes (<em>y</em> axis) as a function of the total number of cell types in which the upregulation occurs. <strong>e</strong>, Gene Ontology terms associated with genes upregulated in the late-pathology group that are common to ≥5 cell types (<em>n</em> = 11 genes, hypergeometric test, FDR correction).</p>
</blockquote>
<h2 id="Fig-3-Trait-associations-in-cellular-subpopulations"><a href="#Fig-3-Trait-associations-in-cellular-subpopulations" class="headerlink" title="Fig. 3: Trait associations in cellular subpopulations."></a>Fig. 3: Trait associations in cellular subpopulations.</h2><p><img src="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig3_HTML.png" alt="Fig. 3"></p>
<blockquote>
<p><strong>a</strong>–<strong>d</strong>, Overrepresentation (hypergeometric test) within each sub-cluster of cells isolated from individuals with varying amyloid levels (<strong>a</strong>), Braak stage (braaksc) (<strong>b</strong>), cognitive status at time of death (cogdx, 1 = no impairment; 4 = impairment) (<strong>c</strong>), and sex (<strong>d</strong>). Only significant associations after FDR correction over all variables and subpopulations are displayed. <strong>e</strong>, Fraction of cells in each sub-cluster. <strong>f</strong>, Quantitative enrichment of clinico-pathological features across sub-clusters for: global AD-pathology burden, neurofibrillary tangle burden; neuritic plaque count, overall amyloid level, and global cognitive function (global_cogn); <em>z</em>-scores were estimated using resampling. The size of the squares is proportional to the numeric entry. <strong>g</strong>, Immunohistochemistry with anti-OLIG2 (red) and anti-CRYAB (green) antibodies in the white matter of Brodmann area 10 of a no-pathology and an AD-pathology individual (scale bars (top left, white), 10 μm). The experiment was performed once. <strong>h</strong>, Similar to <strong>g</strong>, for anti-OLIG2 (red) and anti-QDPR (green). The experiment was performed once.</p>
</blockquote>
<h2 id="Fig-4-Sex-specific-differential-response-to-AD-pathology"><a href="#Fig-4-Sex-specific-differential-response-to-AD-pathology" class="headerlink" title="Fig. 4: Sex-specific differential response to AD pathology."></a>Fig. 4: Sex-specific differential response to AD pathology.</h2><p><img src="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig4_HTML.png" alt="Fig. 4"></p>
<blockquote>
<p>Individual-level transcriptome-wide gene–trait correlation analysis. Box plots show the distribution of correlation values (Pearson correlation coefficient) computed between gene-expression profiles (<em>n</em> = 17,926 genes) averaged for cells of each type across each individual, and the corresponding pathological measurements across individuals (<em>n</em> = 24 female individuals and <em>n</em> = 24 male individuals). Box plots are centred around the median, with the interquartile range (IQR) defining the box. The upper whisker extends to the largest value no further than 1.5 × IQR from the end of the box. The lower whisker extends to the smallest value at most 1.5 × IQR from the end of the box. Pathological traits are represented in red font and cognition in blue. Rectangles around plots highlight the most contrasting differences observed between male and female transcriptional responses; blue indicates a dominant positive correlation and red a dominant negative correlation.</p>
</blockquote>
<h2 id="Extended-Data-Fig-2-snRNA-seq-profiling-and-cell-type-characterization"><a href="#Extended-Data-Fig-2-snRNA-seq-profiling-and-cell-type-characterization" class="headerlink" title="Extended Data Fig. 2: snRNA-seq profiling and cell-type characterization."></a>Extended Data Fig. 2: snRNA-seq profiling and cell-type characterization.</h2><p><img src="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig6_ESM.jpg" alt="Extended Data Fig. 2"></p>
<blockquote>
<p><strong>a</strong>, Study cohort and sample preparation. <strong>b</strong>, Clustering analysis workflow. <strong>c</strong>, Two-dimensional <em>t</em>-SNE projection of all annotated cells (<em>n</em> = 75,060 from 24 pathology and 24 no-pathology individuals). <strong>d</strong>, Correlation matrix (Pearson correlation coefficient; pcc) of the average expression profiles by cell type for each individual.</p>
</blockquote>
<h2 id="Extended-Data-Fig-3-Consistency-of-cells-of-the-same-type-across-individuals"><a href="#Extended-Data-Fig-3-Consistency-of-cells-of-the-same-type-across-individuals" class="headerlink" title="Extended Data Fig. 3: Consistency of cells of the same type across individuals."></a>Extended Data Fig. 3: Consistency of cells of the same type across individuals.</h2><p><img src="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig7_ESM.jpg" alt="Extended Data Fig. 3"></p>
<blockquote>
<p><strong>a</strong>, Expression of known cell-type marker genes for each cell type. <strong>b</strong>, Left, expression of known cell-type marker genes in each cluster; right, fraction of cells in each cluster that express each marker gene. Vertical dashed blue line represents a scale bar referencing 0.5. <strong>c</strong>, Overrepresentation analysis (hypergeometric test) within each of the pre-cluster marker sets (rows) of genes previously identified as markers<a target="_blank" rel="noopener" href="https://www.nature.com/articles/s41586-019-1195-2#ref-CR18">18</a> (columns, left; <em>n</em> = 1,729 total genes) and genes previously identified as markers of cortical layers<a target="_blank" rel="noopener" href="https://www.nature.com/articles/s41586-019-1195-2#ref-CR21">21</a> (columns, right; <em>n</em> = 3,400 total genes). <strong>d</strong>, Fraction of cells of each type isolated across all (<em>n</em> = 48), no-pathology (<em>n</em> = 24), and AD-pathology (<em>n</em> = 24) individuals. <strong>e</strong>, Fraction of cells of each type isolated from each individual (columns; <em>n</em> = 48).</p>
</blockquote>
<h2 id="Extended-Data-Fig-4-Expression-values-and-validation-of-top-DEGs"><a href="#Extended-Data-Fig-4-Expression-values-and-validation-of-top-DEGs" class="headerlink" title="Extended Data Fig. 4: Expression values and validation of top DEGs."></a>Extended Data Fig. 4: Expression values and validation of top DEGs.</h2><p><img src="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig8_ESM.jpg" alt="Extended Data Fig. 4"></p>
<blockquote>
<p><strong>a</strong>, Mean expression values of genes across the nuclei isolated from each individual. Each point represents one individual. DEGs were classified as low, mid, or high in expression, based on their median expression level across the cells of the corresponding cell type. Groups were defined based on <em>k</em>-means clustering (<em>k</em> = 3). The top three genes for each group (low-, mid-, and high-expression levels) and for each cell type are shown. For oligodendrocyte precursor cells and microglia, only one and two genes, respectively, were classified within the high-expression group. <em>ATPIF1</em> is also known as <em>ATP5IF1</em>, <em>NGFRAP1</em>is also known as <em>BEX3</em>, <em>TMEM2</em> is also known as <em>CEMIP2</em>, and <em>ERBB2IP</em> is also known as <em>ERBIN</em>. <strong>b</strong>, Left, RNA in situ hybridization (RNAscope) with probes that detect the excitatory neuron marker <em>SLC17A7</em> (red) and <em>NTNG1</em> (blue) in the grey matter of Brodmann area 10 of a no-pathology and an AD-pathology individual. The tissue was counterstained with haematoxylin. Right, quantification of RNA in situ hybridization on Brodmann area 10 tissue sections. Data are mean ± s.e.m.; *<em>P</em> = 0.047 (Student’s two-tailed <em>t</em>-test). <em>n</em> = 4 no-pathology and <em>n</em> = 4 AD-pathology individuals; <em>n</em> = 5 or 6 images per individual.</p>
</blockquote>
<h2 id="Extended-Data-Fig-5-Overlap-of-genes-that-are-altered-in-the-progression-of-AD-pathology"><a href="#Extended-Data-Fig-5-Overlap-of-genes-that-are-altered-in-the-progression-of-AD-pathology" class="headerlink" title="Extended Data Fig. 5: Overlap of genes that are altered in the progression of AD pathology."></a>Extended Data Fig. 5: Overlap of genes that are altered in the progression of AD pathology.</h2><p><img src="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig9_ESM.jpg" alt="Extended Data Fig. 5"></p>
<blockquote>
<p>Quantification of the overlap (Jaccard coefficient) between pairs of gene sets identified as differentially expressed in each of the major cell types when comparing cells isolated from AD-pathology individuals with cells isolated from no-pathology individuals, and combinations of early- and late-pathology individuals.</p>
</blockquote>
<h2 id="Extended-Data-Fig-7-Overrepresentation-analyses-for-cells-in-sub-clusters"><a href="#Extended-Data-Fig-7-Overrepresentation-analyses-for-cells-in-sub-clusters" class="headerlink" title="Extended Data Fig. 7: Overrepresentation analyses for cells in sub-clusters."></a>Extended Data Fig. 7: Overrepresentation analyses for cells in sub-clusters.</h2><p><img src="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig11_ESM.jpg" alt="Extended Data Fig. 7"></p>
<blockquote>
<p><strong>a</strong>, Cell composition of each identified sub-cluster (rows) across individuals (columns). Bars represent the fraction of cells corresponding to each individual. Bar colour indicates whether the corresponding value exceeds (black) or does not exceed (white) the average value measured across all the entries in the row. <strong>b</strong>, Overrepresentation analysis (hypergeometric test) within each pre-cluster (columns) of cells isolated from each individual (rows). <strong>c</strong>, Overrepresentation analysis within each sub-cluster of cells isolated from individuals with different values of discrete clinico-pathological variables (overall amyloid level, Braak stage, CERAD score (ceradsc), NIA–Reagan score (niareagansc), clinical consensus diagnosis of cognitive status at the time of death (cogdx), and sex). The scale bars on the right indicate the significance of the overrepresentation (hypergeometric test, −log10(<em>P</em> value), <em>z</em>-scaled, FDR multiple-testing correction). For quantitative variables, enrichment was computed based on an estimated <em>z</em>-score quantifying the deviation from random expected values using resampling (<a target="_blank" rel="noopener" href="https://www.nature.com/articles/s41586-019-1195-2#Sec10">Methods</a>). The quantitative variables considered were neuritic plaque count, neurofibrillary tangle burden, tangle density, overall amyloid level, and global cognitive function. For a detailed description of clinico-pathological variables, see <a target="_blank" rel="noopener" href="https://www.nature.com/articles/s41586-019-1195-2#MOESM1">Supplementary Information</a>. <strong>d</strong>, Overrepresentation analysis (hypergeometric test) similar to that in <strong>a</strong>, but computed only across cells isolated from randomly chosen female and male individuals for AD-pathology and no-pathology groups (<a target="_blank" rel="noopener" href="https://www.nature.com/articles/s41586-019-1195-2#Sec10">Methods</a>). Scores represent aggregated <em>P</em> values (meta-p values, meanp method, metap R package) computed across 100 random realizations. Only scores with a FDR &lt; 0.01 (correction across traits × subpopulations) are plotted.</p>
</blockquote>
<h2 id="Extended-Data-Fig-8-Cell-type-subpopulations"><a href="#Extended-Data-Fig-8-Cell-type-subpopulations" class="headerlink" title="Extended Data Fig. 8: Cell-type subpopulations."></a>Extended Data Fig. 8: Cell-type subpopulations.</h2><p><img src="http://maxbond233.oss-cn-shanghai.aliyuncs.com/img/41586_2019_1195_Fig12_ESM.jpg" alt="Extended Data Fig. 8"></p>
<blockquote>
<p>Cells from sub-clusters enriched (red) or depleted (blue) with cells for individuals with AD pathology and cognitive decline shown using <em>t</em>-SNE for major cell types (Ast1 <em>n</em> = 1,134, Ast0 <em>n</em> = 1,728, Oli0 <em>n</em> = 8,310, Oli1 <em>n</em> = 8,032, Ex4 <em>n</em> = 3,198, Ex6 <em>n</em> = 2,757, In0 <em>n</em> = 2,368, In2 <em>n</em> = 984, Opc0 <em>n</em> = 1,589, Opc1 <em>n</em> = 976, Mic1 <em>n</em> = 509, and Mic2 <em>n</em> = 169 cells). Left, corresponding marker genes (font proportional to enrichment level); right, enriched Gene Ontology terms. Gene Ontology enrichment was based on FDR-corrected cumulative hypergeometric <em>P</em> values, with <em>P</em> value-ranked gene-marker lists (FDR &lt; 0.01, log2(mean gene expression across cells in sub-cluster/mean gene expression across cells in other sub-clusters) &gt; 0.5, two-sided Wilcoxon rank-sum test) used as input (Ex4 <em>n</em> = 783, Ex6 <em>n</em> = 2,438, In0 <em>n</em> = 1,702, In2 <em>n</em> = 350, Ast1 <em>n</em> = 574, Ast0 <em>n</em> = 73, Oli0 <em>n</em> = 227, Oli1 <em>n</em> = 73, Opc0 <em>n</em> = 19, Opc1 <em>n</em> = 536, Mic1 <em>n</em> = 487, and Mic2 <em>n</em> = 646).</p>
</blockquote>
<p>reference:</p>
<ul>
<li><a target="_blank" rel="noopener" href="https://www.nature.com/articles/s41586-019-1195-2">Single-cell transcriptomic analysis of Alzheimer’s disease</a></li>
</ul>

    </div>

    
    
    

      <footer class="post-footer">
          <div class="post-tags">
              <a href="/tags/single-cell-RNA-seq/" rel="tag"># single-cell RNA-seq</a>
              <a href="/tags/brain/" rel="tag"># brain</a>
              <a href="/tags/prefrontal-cortex/" rel="tag"># prefrontal cortex</a>
              <a href="/tags/disease/" rel="tag"># disease</a>
              <a href="/tags/AD/" rel="tag"># AD</a>
          </div>

        


        
    <div class="post-nav">
      <div class="post-nav-item">
    <a href="/2021/12/04/%E5%8D%95%E7%BB%86%E8%83%9E%E8%BD%AC%E5%BD%95%E7%BB%84%E6%96%87%E7%8C%AE%E5%88%86%E6%9E%90%E6%96%B9%E6%B3%95-%E4%BA%8C-Single-cell-transcriptomic-analysis-of-the-lateral-hypothalamic-area-reveals-molecularly-distinct-populations-of-inhibitory-and-excitatory-neurons/" rel="prev" title="单细胞转录组文献分析方法[二]: Single-cell transcriptomic analysis of the lateral hypothalamic area reveals molecularly distinct populations of inhibitory and excitatory neurons">
      <i class="fa fa-chevron-left"></i> 单细胞转录组文献分析方法[二]: Single-cell transcriptomic analysis of the lateral hypothalamic area reveals molecularly distinct populations of inhibitory and excitatory neurons
    </a></div>
      <div class="post-nav-item">
    <a href="/2021/12/13/%E5%8D%95%E7%BB%86%E8%83%9E%E8%BD%AC%E5%BD%95%E7%BB%84%E6%96%87%E7%8C%AE%E5%88%86%E6%9E%90%E6%96%B9%E6%B3%95-%E5%9B%9B-Human-and-mouse-single-nucleus-transcriptomics-reveal-TREM2-dependent-and-TREM2-independent-cellular-responses-in-Alzheimer%E2%80%99s-disease/" rel="next" title="单细胞转录组文献分析方法[四]:Human and mouse single-nucleus transcriptomics reveal TREM2-dependent and TREM2-independent cellular responses in Alzheimer’s disease">
      单细胞转录组文献分析方法[四]:Human and mouse single-nucleus transcriptomics reveal TREM2-dependent and TREM2-independent cellular responses in Alzheimer’s disease <i class="fa fa-chevron-right"></i>
    </a></div>
    </div>
      </footer>
    
  </article>
  
  
  



          </div>
          

<script>
  window.addEventListener('tabs:register', () => {
    let { activeClass } = CONFIG.comments;
    if (CONFIG.comments.storage) {
      activeClass = localStorage.getItem('comments_active') || activeClass;
    }
    if (activeClass) {
      let activeTab = document.querySelector(`a[href="#comment-${activeClass}"]`);
      if (activeTab) {
        activeTab.click();
      }
    }
  });
  if (CONFIG.comments.storage) {
    window.addEventListener('tabs:click', event => {
      if (!event.target.matches('.tabs-comment .tab-content .tab-pane')) return;
      let commentClass = event.target.classList[1];
      localStorage.setItem('comments_active', commentClass);
    });
  }
</script>

        </div>
          
  
  <div class="toggle sidebar-toggle">
    <span class="toggle-line toggle-line-first"></span>
    <span class="toggle-line toggle-line-middle"></span>
    <span class="toggle-line toggle-line-last"></span>
  </div>

  <aside class="sidebar">
    <div class="sidebar-inner">

      <ul class="sidebar-nav motion-element">
        <li class="sidebar-nav-toc">
          文章目录
        </li>
        <li class="sidebar-nav-overview">
          站点概览
        </li>
      </ul>

      <!--noindex-->
      <div class="post-toc-wrap sidebar-panel">
          <div class="post-toc motion-element"><ol class="nav"><li class="nav-item nav-level-1"><a class="nav-link" href="#Abstract"><span class="nav-number">1.</span> <span class="nav-text">Abstract</span></a></li><li class="nav-item nav-level-1"><a class="nav-link" href="#Methods"><span class="nav-number">2.</span> <span class="nav-text">Methods</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#Cell-clustering"><span class="nav-number">2.1.</span> <span class="nav-text">Cell clustering</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Cell-type-annotation-and-sub-clustering"><span class="nav-number">2.2.</span> <span class="nav-text">Cell type annotation and sub-clustering</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Marker-identification"><span class="nav-number">2.3.</span> <span class="nav-text">Marker identification</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Differential-gene-expression-analysis"><span class="nav-number">2.4.</span> <span class="nav-text">Differential gene-expression analysis</span></a></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#Figure"><span class="nav-number">3.</span> <span class="nav-text">Figure</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#Fig-1-Cell-type-specific-gene-expression-changes-in-AD-pathology"><span class="nav-number">3.1.</span> <span class="nav-text">Fig. 1: Cell-type-specific gene-expression changes in AD pathology.</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Fig-2-Gene-expression-changes-in-the-progression-of-AD-pathology"><span class="nav-number">3.2.</span> <span class="nav-text">Fig. 2: Gene-expression changes in the progression of AD pathology.</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Fig-3-Trait-associations-in-cellular-subpopulations"><span class="nav-number">3.3.</span> <span class="nav-text">Fig. 3: Trait associations in cellular subpopulations.</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Fig-4-Sex-specific-differential-response-to-AD-pathology"><span class="nav-number">3.4.</span> <span class="nav-text">Fig. 4: Sex-specific differential response to AD pathology.</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Extended-Data-Fig-2-snRNA-seq-profiling-and-cell-type-characterization"><span class="nav-number">3.5.</span> <span class="nav-text">Extended Data Fig. 2: snRNA-seq profiling and cell-type characterization.</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Extended-Data-Fig-3-Consistency-of-cells-of-the-same-type-across-individuals"><span class="nav-number">3.6.</span> <span class="nav-text">Extended Data Fig. 3: Consistency of cells of the same type across individuals.</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Extended-Data-Fig-4-Expression-values-and-validation-of-top-DEGs"><span class="nav-number">3.7.</span> <span class="nav-text">Extended Data Fig. 4: Expression values and validation of top DEGs.</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Extended-Data-Fig-5-Overlap-of-genes-that-are-altered-in-the-progression-of-AD-pathology"><span class="nav-number">3.8.</span> <span class="nav-text">Extended Data Fig. 5: Overlap of genes that are altered in the progression of AD pathology.</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Extended-Data-Fig-7-Overrepresentation-analyses-for-cells-in-sub-clusters"><span class="nav-number">3.9.</span> <span class="nav-text">Extended Data Fig. 7: Overrepresentation analyses for cells in sub-clusters.</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Extended-Data-Fig-8-Cell-type-subpopulations"><span class="nav-number">3.10.</span> <span class="nav-text">Extended Data Fig. 8: Cell-type subpopulations.</span></a></li></ol></li></ol></div>
      </div>
      <!--/noindex-->

      <div class="site-overview-wrap sidebar-panel">
        <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
  <p class="site-author-name" itemprop="name">Max</p>
  <div class="site-description" itemprop="description">在学了在学了</div>
</div>
<div class="site-state-wrap motion-element">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
          <a href="/archives/">
        
          <span class="site-state-item-count">11</span>
          <span class="site-state-item-name">日志</span>
        </a>
      </div>
      <div class="site-state-item site-state-categories">
            <a href="/categories/">
          
        <span class="site-state-item-count">3</span>
        <span class="site-state-item-name">分类</span></a>
      </div>
      <div class="site-state-item site-state-tags">
            <a href="/tags/">
          
        <span class="site-state-item-count">14</span>
        <span class="site-state-item-name">标签</span></a>
      </div>
  </nav>
</div>



      </div>

    </div>
  </aside>
  <div id="sidebar-dimmer"></div>


      </div>
    </main>

    <footer class="footer">
      <div class="footer-inner">
        

        

<div class="copyright">
  
  &copy; 
  <span itemprop="copyrightYear">2022</span>
  <span class="with-love">
    <i class="fas fa-bug"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">Max</span>
</div>
  <div class="powered-by">由 <a href="https://hexo.io/" class="theme-link" rel="noopener" target="_blank">Hexo</a> & <a href="https://pisces.theme-next.org/" class="theme-link" rel="noopener" target="_blank">NexT.Pisces</a> 强力驱动
  </div>

        








      </div>
    </footer>
  </div>

  
  <script src="/lib/anime.min.js"></script>
  <script src="/lib/velocity/velocity.min.js"></script>
  <script src="/lib/velocity/velocity.ui.min.js"></script>

<script src="/js/utils.js"></script>

<script src="/js/motion.js"></script>


<script src="/js/schemes/pisces.js"></script>


<script src="/js/next-boot.js"></script>




  















  

  

</body>
</html>
